templates/actors/parts/columns/column-feature-controls.hbs

```diff
@@ -0,0 +1,33 @@
+<div class="item-detail item-controls">
+
+    {{#if @root.editable}}
+    {{!-- Editing --}}
+    <a class="item-control item-action" data-action="edit" data-tooltip="DND5E.ItemEdit"
+       aria-label="{{ localize "DND5E.ItemEdit" }}">
+        <i class="fas fa-pen-to-square"></i>
+    </a>
+
+    {{!-- Deleting --}}
+    <a class="item-control item-action" data-action="delete" data-tooltip="DND5E.ItemDelete"
+       aria-label="{{ localize "DND5E.ItemDelete" }}">
+        <i class="fas fa-trash"></i>
+    </a>
+    {{/if}}
+
+    {{!-- Expand/Collapse --}}
+    <a class="item-control interface-only" data-toggle-description
```

Comment:
Not hit box of the context menu button seems to overlapping the expand/collapse button making it hard to click in edit mode or on the inventory tab:

<img width="88" alt="Screenshot 2024-07-04 at 09 19 01" src="https://github.com/foundryvtt/dnd5e/assets/19979839/d326969d-7a97-4348-859d-7f664e2f2dc2">